mozregression/main.py

```diff
@@ -124,9 +124,9 @@ def parse_args(argv=None):
 
     parser.add_argument("-c", "--command",
                         help=("Test command to evaluate builds automatically."
-                              " A return code of 0 will evaluate build as good,"
-                              " any other value will evaluate the build as"
-                              " bad."))
+                              " A return code of 0 will evaluate build as "
+                              "good, any other value will evaluate the build"
```

Comment:
Could you put the space before good, instead of after the as ? This will be consistent with the rest of the code.